Description: Move doc8 to test requirements
 doc8 is not required to use this module, only for style
 checking of RST documentation.
Author: James Page <james.page@canonical.com>
Forwarded: https://review.openstack.org/#/c/591664/

--- a/glance_store.egg-info/requires.txt
+++ b/glance_store.egg-info/requires.txt
@@ -4,7 +4,6 @@ oslo.serialization!=2.19.1,>=2.18.0
 oslo.utils>=3.33.0
 oslo.concurrency>=3.26.0
 stevedore>=1.20.0
-doc8>=0.6.0
 eventlet!=0.18.3,!=0.20.1,>=0.18.2
 six>=1.10.0
 jsonschema<3.0.0,>=2.6.0
--- a/requirements.txt
+++ b/requirements.txt
@@ -8,7 +8,6 @@ oslo.utils>=3.33.0 # Apache-2.0
 oslo.concurrency>=3.26.0 # Apache-2.0
 stevedore>=1.20.0 # Apache-2.0
 enum34>=1.0.4;python_version=='2.7' or python_version=='2.6' or python_version=='3.3' # BSD
-doc8>=0.6.0 # Apache-2.0
 eventlet!=0.18.3,!=0.20.1,>=0.18.2 # MIT
 six>=1.10.0 # MIT
 
